src/scene.ts

```diff
@@ -2585,16 +2578,8 @@ export class Scene extends AbstractScene implements IAnimatable {
             Frustum.GetPlanesToRef(this._transformMatrix, this._frustumPlanes);
         }
 
-        if (viewR && projectionR) {
-            viewR.multiplyToRef(projectionR, Tmp.Matrix[0]);
-            Frustum.GetRightPlaneToRef(Tmp.Matrix[0], this._frustumPlanes[3]); // Replace right plane by second camera right plane
-        }
-
         if (this._multiviewSceneUbo && this._multiviewSceneUbo.useUbo) {

```

Comment:
Could this be done by callbacks like the other components instead of the test ?

Reply:
I am finding using the observables a bit tricky as for multiview it's not really doing something extra, it is doing something different instead. I could add something like a renderMultiviewToSingleViewObservable but then it's not any different than a function call.